### ISSUE TITLE
Explanation argument

### ISSUE THREAD

**Issue by sillasgonzaga** [timelyportfolio/sunburstR#6]
I can't seem to use a user defined JS function in the explanation that returns d.name and d.percent, but the d object does not have a percent value. For example:

`sunburst(sequence_data, explanation = "function(d) {return [d.name, d.value]}")`

The code above works, but this one below not:

```
> (x <- sum(sequence_data$V2))
[1] 41768062
> sunburst(sequence_data, explanation = "function(d) {return d.value / x}")
```

Is there another way to show d.name and percent in the output

**Comment by timelyportfolio** [comment]
If I understand your question correctly, the `htmlwidget`/`JavaScript` will have no way of seeing `x` unless it is provided within the function.  It might be helpful if I understood the objective.   These [lines](https://github.com/timelyportfolio/sunburstR/blob/8845323c292e4eb6416ed133e2e2f7117bc42935/inst/htmlwidgets/sunburst.js#L150-L153)  I just changed so that we bind `totalSize`.  `totalSize` will now be available as `this` within our `explanation` function.  Perhaps, this new example will be helpful.

```
# reinstall newest
# devtools::install_github("timelyportfolio/sunburstR")
sunburst(
  sequences,
  # note this is html, so you can even style this
  #  as you would normal html
  explanation = "
function(d) {
  return 'everything sum = ' + this + '<br> d = ' + d.value
}
  "
)
```

**Comment by sillasgonzaga** [comment]
I believe that the fix you provided for #7 solves the problem I described here. Thanks.
